src/widget/form/addfriendform.h

```diff
@@ -55,9 +55,9 @@ class AddFriendForm : public QObject
     bool addFriendRequest(const QString& friendAddress, const QString& message);
 
 signals:
-    void friendRequested(const QString& friendAddress, const QString& message);
-    void friendRequestAccepted(const QString& friendAddress);
-    void friendRequestsSeen();
+    void friendRequested(const QString& friendAddress, const QString& message) const;
+    void friendRequestAccepted(const QString& friendAddress) const;
+    void friendRequestsSeen() const;
```

Comment:
Unfortunately, it make. We can't emit non-const signal from const context (don't know why)